apps/api/src/providers/secrets/index.ts

```diff
@@ -0,0 +1,39 @@
+import { EmailProviderIdEnum } from '@novu/shared';
+import axios from 'axios';
+
+const getDopplerSecrets = async () => {
```

Comment:
What do you think about the following:
Instead of fetching doppler secrets in code we can just rely on the env variables to be present. And run the doppler CLI `run` command when running the tests. It will populate all env variables automatically for the run project. In this way we benefit The ability that our code doesn't know about doppler, we can run this code with env variables from GitHub actions, ECS, Doopler and etc...

so we can later easily switch where the variables are coming from. 

https://docs.doppler.com/docs/install-cli#usage

Reply:
I was considering that possibility but the need of having to log in to run the Doppler CLI pushed me back as it would be annoying for any user having to have their own Doppler account. And for me still makes the code to be aware of Doppler existence in a tighter way. Because of this one of the reasons of going with the API is the ability of quickly revoke the Doppler access token if compromised, thus we should only have an entry vector. Compared with the login that we would have any member added to the Doppler organisation.

Reply:
Got ya, that makes sense. Thanks for the explanation 🙏